plugins/generator-1.15.2/forge-1.15.2/templates/block.java.ftl

```diff
@@ -258,71 +259,16 @@ public class ${name}Block extends ${JavaModName}Elements.ModElement {
 		}
 		</#if>
 
-		<#if data.mx != 0 || data.my != 0 || data.mz != 0 || data.Mx != 1 || data.My != 1 || data.Mz != 1>
+		<#if data.boundingBoxes?? && data.allowsCustomBoundingBox() && !data.isFullCube()>

```

Comment:
The check `data.boundingBoxes??` makes the system work so that if no bounding boxes are specified on the list, the default box will be used. I am not sure if this is the desired behavior as probably if the list is empty, the user does not want to add any bounding boxes. The same applies to plants/blocks of both versions.

Reply:
Disregard this, ?? is for null check only and not for empty lists :)